app/views/import_wizards/index.haml

```diff
@@ -41,9 +49,17 @@
     -else
       = link_to _('Download a template CSV file'), sample_csv_api_collection_path(collection, format: 'csv'), class: "icon fimport black"
 
-    %form#upload_form{action: upload_csv_collection_import_wizard_path(collection), method: :post, enctype: 'multipart/form-data'}
-      %input{type: :hidden, name: 'authenticity_token', value: form_authenticity_token}
-      .icon.fexport.black Upload a CSV file to update multiple sites
-      %input#upload{type: :file, name: :file}
+      %div
+        %form#upload_form{action: upload_csv_collection_import_wizard_path(collection), method: :post, enctype: 'multipart/form-data', style: 'display: inline-block'}
+          %input{type: :hidden, name: 'authenticity_token', value: form_authenticity_token}
+          %div
+            .icon.fexport.black Upload a CSV file to update multiple sites
+            %input#upload{type: :file, name: :file, style: 'visibility:hidden; width:1px; height: 1px'}
+        %form#upload_form_spreadsheet{action: import_csv_from_google_spreadsheet_collection_import_wizard_path(collection), method: :post, enctype: 'multipart/form-data', style: 'display: inline-block'}
+          %input{type: :hidden, name: 'authenticity_token', value: form_authenticity_token}
+          .icon.syes_no.black{style: 'cursor: pointer;'} or add a spreadsheet url
+          %input{type: 'text', id: :spreadSheetLink, name: 'spreadSheetLink', placeholder: 'Enter url', style: 'width: 300px;'}
```

Comment:
I was not very sure where to put these css properties since there are not a lot stylesheets files. I know inline styles are not a good practice but since they were being used across several places in the application, I kind of followed the same rule.

Reply:
It seems to be ResourceMap's way - it's OK 👍